Cargo.toml

```diff
@@ -29,86 +29,170 @@ license = "BSD-3-Clause-Clear"
+assert_cmd = "=2.0.17"  # Assert command for testing - LOW RISK: assert-rs team, 1M+ downloads
+async-std = { version = "=1.13.1", features = ["attributes", "tokio1"] }  # Async runtime for testing - LOW RISK: async-rs team, 10M+ downloads
+async-trait = "=0.1.88"  # Async trait support - LOW RISK: dtolnay, widely used
+async_cell = "0.2.2"  # Async cell implementation - MEDIUM RISK: Individual maintainer, very low popularity, consider alternatives
+attestation-doc-validation = { version = "=0.10.0" }  # AWS Nitro attestation validation - LOW RISK: Evervault (reputable security company), security-critical but trusted
+aws-config = { version = "=1.5.9" }  # AWS SDK configuration - LOW RISK: Official AWS SDK, actively maintained
+aws-nitro-enclaves-nsm-api = { version = "=0.4.0" }  # AWS Nitro Enclaves NSM API - MEDIUM RISK: Security-critical enclave operations, needs audit
+aws-sdk-kms = { version = "=1.50.0" }  # AWS KMS client - LOW RISK: Official AWS SDK for key management
+aws-sdk-s3 = { version = "=1.62.0" }  # AWS S3 client - LOW RISK: Official AWS SDK for object storage
+aws-smithy-runtime = { version = "=1.7.3", features = ["client", "connector-hyper-0-14-x"] }  # AWS Smithy runtime - LOW RISK: Official AWS runtime library
+aws-smithy-runtime-api = { version = "=1.7.3" }  # AWS Smithy runtime API - LOW RISK: Official AWS runtime API
+aws-smithy-types = { version = "=1.2.9" }  # AWS Smithy types - LOW RISK: Official AWS type definitions
+axum = { version = "=0.8.4", features = ["tokio"] }  # Web framework - LOW RISK: tokio-rs team, 10M+ downloads, actively maintained
+backoff = "=0.4.0"  # Retry with exponential backoff - LOW RISK: Popular utility
+base64 = "=0.22.1"  # Base64 encoding - LOW RISK: marshallpierce, 100M+ downloads
+bincode = { version = "=2.0.1", features = ["serde"] }  # Binary serialization - LOW RISK: bincode-org, 20M+ downloads
+bip39 = { version = "=2.2.0", features = ["alloc"] }  # BIP39 mnemonic seed phrases - LOW RISK: rust-bitcoin org (reputable), 1M+ downloads, 10+ developers, CI enabled
+bytes = "=1.10.1"  # Byte buffer utilities - LOW RISK: tokio team, 100M+ downloads
+cbc = { version = "=0.1.2", features = ["alloc"] }  # CBC block cipher mode - LOW RISK: RustCrypto org, standard cipher mode, 5M+ downloads
+cfg-if = "=1.0.3"  # Conditional compilation - LOW RISK: alexcrichton, 100M+ downloads
+clap = { version = "=4.5.47", features = ["derive"] }  # CLI argument parsing - LOW RISK: clap-rs team, 100M+ downloads
+config = "=0.15.15"  # Configuration management - LOW RISK: mehcode, popular
+const_format = "=0.2.34"  # Compile-time string formatting - LOW RISK: rodrimati1992, 1M+ downloads
+console_error_panic_hook = { version = "=0.1.7" }  # WASM panic hook for better error messages - LOW RISK: rustwasm team
+criterion = { version = "=0.5.1", features = ["async_tokio"] }  # Benchmarking framework - LOW RISK: bheisler, 5M+ downloads
```

Comment:
Single owner, very inactive owner for many years (at least when it comes to commits). I would put this at least in MEDIUM, maybe even HIGH

Reply:
Agree as per https://github.com/zama-ai/kms/pull/132#discussion_r2359611056

Reply:
@dd23 I used some automation for statistics gathering and it appeared to be outdated a bit. So manually reverifying everything. A single owner because I looked at GitHub repos. See that it's a private repo but not of a team or org repo 
<img width="319" height="89" alt="image" src="https://github.com/user-attachments/assets/bc0e4f9c-3a83-4a8e-a43e-e4f5b4f9dcf7" />

Reply:
Addressed in https://github.com/zama-ai/kms/pull/132/commits/7eeb41dd26969d7b52a4654e15c6ea7ee0530621